projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/StaticRoute.java

```diff
@@ -110,4 +110,35 @@ protected final String protocolRouteString() {
   public int routeCompare(AbstractRoute rhs) {
     return 0;
   }
+
+  public static class Builder extends AbstractRouteBuilder<StaticRoute> {
+
+    private int _administrativeCost;
+
+    private String _nextHopInterface;
+
+    @Override
+    public StaticRoute build() {
+      if (_nextHopIp == null) {
+        _nextHopIp = Route.UNSET_ROUTE_NEXT_HOP_IP;
+      }
+      return new StaticRoute(_network, _nextHopIp, _nextHopInterface, _administrativeCost, _tag);
+    }
+
+    public int getAdministrativeCost() {
+      return _administrativeCost;
+    }
+
+    public void setAdministrativeCost(int administrativeCost) {
+      this._administrativeCost = administrativeCost;
+    }
+
+    public String getNextHopInterface() {
+      return _nextHopInterface;
+    }
+
+    public void setNextHopInterface(String nextHopInterface) {
```

Comment:
check for invalid values. (Specifically, `null` nextHopInterface should throw an exception).